lib/osf-components/addon/components/addon-card/template.hbs

```diff
@@ -0,0 +1,85 @@
+<div
+    local-class='card-wrapper'
+    data-test-addon-card={{@addon.provider.name}}
+    data-analytics-scope={{concat 'Addon card' @addon.provider.name}}
+>
+    <img
+        data-test-addon-card-logo
+        src={{this.assetLogo}}
+        alt={{t 'osf-components.addon-card.provider-logo-alt' provider=@addon.provider.name}}
+        local-class='addon-card-logo'
+    />
+    <div
+        data-test-addon-card-title
+        local-class='provider-name'
+    >
+        {{@addon.provider.name}}
+    </div>
+    
+    <div local-class='buttons-wrapper'>
+        {{#if this.addonIsConfigured}}
+            <OsfLink
+                data-test-addon-card-configure
+                data-analytics-name='Configure'
+                local-class='button-looking-links configure'
+                @route='guid-node.addons.addon.configure'
+                @models={{array @manager.node.id @addon.provider.id}}
+            >
+                {{t 'osf-components.addon-card.configure'}}
+            </OsfLink>
+            <Button
+                data-test-addon-card-disable
+                data-analytics-name='Disable'
+                local-class='disable'
+                {{on 'click' this.toggleDeleteModal}}
```

Comment:
I'm not seeing another use of toggle, and I don't think this is clickable while the dialog is open. Would it make more sense to just `mut` the property to `true` here rather than having the function? This is super-minor.